src/components/smart_objects/src/object_schema_item.cc

```diff
@@ -99,14 +147,61 @@ Errors::eType CObjectSchemaItem::validate(const SmartObject& object,
   return Errors::OK;
 }
 
-void CObjectSchemaItem::applySchema(SmartObject& Object,
-                                    const bool RemoveFakeParameters) {
+Errors::eType CObjectSchemaItem::validate(
+    const SmartObject& object,
+    rpc::ValidationReport* report__,
+    const utils::SemanticVersion& MessageVersion) {
```

Comment:
Same here, most of this code is duplicated from the other signature of this function